contribs/freight/src/main/java/org/matsim/contrib/freight/events/FreightServiceEndEventCreator.java

```diff
@@ -19,31 +19,28 @@
  *
  */
 
-package org.matsim.contrib.freight.controler;
+package org.matsim.contrib.freight.events;
 
-import org.matsim.api.core.v01.Id;
 import org.matsim.api.core.v01.events.ActivityEndEvent;
 import org.matsim.api.core.v01.events.Event;
 import org.matsim.api.core.v01.population.Activity;
-import org.matsim.api.core.v01.population.Person;
 import org.matsim.contrib.freight.carrier.Carrier;
+import org.matsim.contrib.freight.carrier.FreightConstants;
 import org.matsim.contrib.freight.carrier.ScheduledTour;
 import org.matsim.contrib.freight.carrier.Tour.ServiceActivity;
 import org.matsim.contrib.freight.carrier.Tour.TourElement;
-import org.matsim.contrib.freight.events.LSPServiceEndEvent;
 
-/*package-private*/  final class LSPServiceEndEventCreator implements LSPEventCreator {
+import java.util.Objects;
+
+/*package-private*/  final class FreightServiceEndEventCreator implements FreightEventCreator {
 
 	@Override
-	public Event createEvent(Event event, Carrier carrier, Activity activity, ScheduledTour scheduledTour,
-			Id<Person> driverId, int activityCounter) {
-		if(event instanceof ActivityEndEvent){
-			ActivityEndEvent endEvent = (ActivityEndEvent) event;
-			if(endEvent.getActType() == "service") {
+	public Event createEvent(Event event, Carrier carrier, Activity activity, ScheduledTour scheduledTour, int activityCounter) {
```

Comment:
You could get rid of one level of indentation
```
if (event instanceof ActivityEndEvent endEvent && FreightConstans.SERVICE.equals(endEvent.getActType()) {
...
}
```
Also, using `String::equals` instad of `Objects::equals` is probably more idiomatic

Reply:
Done :)